src/routes/SafeAppLandingPage/SafeAppLandingPage.tsx

```diff
@@ -0,0 +1,321 @@
+                                <List component="div">
+                                  <ConnectDetails />
+                                </List>
+                              </ClickAwayListener>
+                            </Popper>
+                          )
+                        }
+                      />
+                    </StyledProvider>
+                  </>
+                )}
+              </UserSafeContainer>
+
+              {/* Demo Safe */}
+              <SafeDemoContainer>
+                <Title size={'xs'}>Want to try the app before using it?</Title>
+
+                <BodyImage>
+                  <Img alt="Demo" height={92} src={DemoSvg} />
+                </BodyImage>
+                {safeAppUrl && (
+                  <StyledDemoButton
+                    data-testid={'open-demo-app-link'}
+                    color="primary"
+                    component={Link}
+                    to={`${demoSafeAppsPath}?appUrl=${encodeURI(safeAppUrl)}`}
+                    size="lg"
+                    variant="outlined"
+                  >
+                    Try Demo
+                  </StyledDemoButton>
+                )}
+              </SafeDemoContainer>
+            </ActionsContainer>
+          </>
+        )}
+      </StyledCard>
+    </Container>
+  )
+}
+
+export default SafeAppLandingPage
+
+function getUserSafe() {
+  // TODO: to be implemented in https://github.com/gnosis/safe-react-apps/issues/416
+  return undefined
+}
+
+const Container = styled.main`
+  display: flex;
+  justify-content: center;
+  margin-top: 20px;
+`
+
+const StyledCard = styled(Card)`
+  flex-grow: 1;
+  max-width: 850px;
+  border-radius: 20px;
+  padding: 50px 58px;
+`
+
+const LoaderContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 400px;
+`
+
+const DetailsContainer = styled.div`
+  display: flex;
+`
+
+const SafeIcon = styled.img`
+  width: 90px;
+  height: 90px;
+`
+
+const SafeAppTitle = styled(Title)`
+  margin-top: 0px;
+  margin-bottom: 12px;
+`
+
+const DescriptionContainer = styled.div`
+  padding-left: 66px;
+  flex-grow: 1;
+`
+
+const Separator = styled(Divider)`
+  margin: 32px 0;
+`
+
+const ChainLabel = styled(Text)`
+  color: #b2bbc0;
+`
+
+const ChainsContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+
+  && > div {
+    margin-top: 12px;
+    margin-right: 8px;
+  }
+`
+
+const ActionsContainer = styled.div`
+  display: flex;
+`
+
+const UserSafeContainer = styled.div`
+  flex: 1 0 50%;
+  text-align: center;
+`
+
+const SafeDemoContainer = styled.div`
+  flex: 1 0 50%;
+  text-align: center;
+`
+const StyledProvider = styled.div`
+  width: 300px;
+  height: 56px;
+  margin: 0 auto;
+  border-radius: 8px;
+  border: 2px solid #eeeff0;
+`
+
+const BodyImage = styled.div`
+  margin: 30px 0;
+`
+
+const StyledDemoButton = styled(Button)`
+  border: 2px solid #008c73;
+`
```

Comment:
A lot of the values here are exported from `src/theme/variables.js`. Pleaseimport them from there where appropriate instead of declaring them locally..